### DISCUSSION TITLE
Scenic

### DISCUSSION THREAD

**Discussion by muhusmanguj** [carla-simulator/carla#4918] [Q&A]
Hello everybody 
I am using Scenic language to crete scenarios and I am trying to simulate them in carla simulator. I want to retrieve the scenario information so that I can have this information of scenario like simulation time, speeds and distances stored in some file. I want to do some analysis on the information retrieved later. I have tried carla recording function but it did not wotk with scenic , only empty folder is created. Can you guys help me please . I am new to carla . 

Any help on the topic or reference to some related documentation will be highly appreciated.
Looking forward to your suggestions

Best Regards
Muhammad Usman
MS Student.

**Comment by dx6nxr** [comment]
Have you been able to implement it in some way? I am also trying to export the data into osc format from carla, and the only way I found so far is to extract data from the carla recording, which lacks some information needed for openSCENARIO.  

Best Regards,
Ivan Burmistrov
